### PR TITLE
Initial content for the repo

### LINKED ISSUE #1: Add initial content for repository
**Description:**
Add initial content for the repository.

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**

### CODE REVIEW

File: helloworld/test/HelloWorldTest.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                <assertEquals>
+                    <actual>$body</actual>
+                    <expected><![CDATA[{"Hello":"World"}]]></expected>
+                    <message>Actual payload is not equals to the expected value</message>
+                </assertEquals>
+            </assertions>
+        </test-case>
+    </test-cases>
+    <mock-services/>
+</unit-test>
```
Comment (chirangaalwis) [inline helloworld/test/HelloWorldTest.xml]: @ThilinaManamgoda please add a new EOF line.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ENV MAVEN_OPTS="-Dmaven.repo.local=/root/m2repo/"
+COPY helloworld/pom.xml /usr/src/mymaven/helloworld/
+RUN mvn -f /usr/src/mymaven/helloworld/pom.xml dependency:go-offline -B
+
+COPY helloworld /usr/src/mymaven/helloworld
+COPY helloworldCompositeApplication /usr/src/mymaven/helloworldCompositeApplication
+RUN mvn -f /usr/src/mymaven/helloworld/pom.xml clean install -Dmaven.test.skip=true
+RUN mvn -f /usr/src/mymaven/helloworldCompositeApplication/pom.xml clean install -Dmaven.test.skip=true
+
+FROM <BASE>
```
Comment (chirangaalwis) [inline Dockerfile:30]: @ThilinaManamgoda just out of curiosity, does the `<BASE>` refer to an official WSO2 product Docker image? Is setting this, handled by the pipeline?

Comment (ThilinaManamgoda) [reply Dockerfile:30]: It is handled by the pipeline.

File: helloworld/test/HelloWorldTest.xml (hunk 1 above)
Comment (ThilinaManamgoda) [reply helloworld/test/HelloWorldTest.xml]: done

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**